crate_universe/src/main.rs

```diff
@@ -16,7 +17,9 @@ struct Opt {
     #[structopt(long = "input_path", parse(from_os_str))]
     input_path: PathBuf,
     #[structopt(long = "output_path", parse(from_os_str))]
-    output_path: PathBuf,
+    output_path: Option<PathBuf>,
```

Comment:
Let's support one or the other option here, I don't see value in supporting both. Can you either remove `output_path`, or remove `repository_dir`?

It looks like long term you're going to prefer `repository_dir`, so let's maybe drop `output_path`, and just change what we generate in the `repository_dir` in the future?

Reply:
I was going to remove `output_path` but then all the tests started to fail and I didn't want to go down that rabbit hole just yet. I do think using `repository_dir` is the better thing though and have added comments to indicate this. My plan was to address this in a second PR.

Reply:
In that case, let's maybe drop the `repository_dir` stuff, and fully implement that in one PR, rather than half doing it now? (Sorry, this change already does quite a few things, so it'd be good to avoid half-doing this one :))

Reply:
Yeah, I didn't realize how little `repository_dir` was used after reviving `output_path` for tests. Removed it.